src/openforms/submissions/tests/test_variables/test_fetch_with_logic.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+import requests_mock
+from factory.django import FileField
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from zgw_consumers.constants import APITypes, AuthTypes
+
+from openforms.forms.constants import LogicActionTypes
+from openforms.forms.tests.factories import FormLogicFactory
+from openforms.registrations.contrib.zgw_apis.tests.factories import ServiceFactory
+from openforms.variables.tests.factories import ServiceFetchConfigurationFactory
+
+from ..factories import SubmissionFactory
+from ..mixins import SubmissionsMixin
+
+
+class ServiceFetchWithActionsTest(SubmissionsMixin, APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.service = ServiceFactory.create(
+            api_type=APITypes.orc,
+            api_root="https://httpbin.org/",
+            auth_type=AuthTypes.no_auth,
+            oas_file=FileField(
+                from_path=str(Path(__file__).parent.parent / "files" / "openapi.yaml")
+            ),
+        )
+
+    @requests_mock.mock(case_sensitive=True)
```

Comment:
```suggestion
    @requests_mock.Mocker(case_sensitive=True)
```
they're the same, but for consistency's sake in the codebase we prefer this variant.